Plots/Masking/NCL_lcmask_1.py

```diff
@@ -0,0 +1,147 @@
+   - Zooming in on a particular area on a Lambert Conformal map
+   - Creating a custom plot boundary
+   - Using a blue-white-red color map
+   - Setting contour levels using a min/max contour level and a spacing
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: http://ncl.ucar.edu/Applications/Scripts/lcmask_1.ncl
+    - Original NCL plot: http://ncl.ucar.edu/Applications/Images/lcmask_1_1_lg.png and http://ncl.ucar.edu/Applications/Images/lcmask_1_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import cartopy.crs as ccrs
+import matplotlib.path as mpath
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Utility function to create a wedge shaped path for plot boundary:
+def wedge_boundary(ax, lon_range, lat_range, res=1):
+    """
+    Utility function to create a custom wedge shaped map boundary using given
+    ranges of longitudes and latitudes.
+
+    Args:
+
+        ax (:class:'matplotlib.axes'):
+            The axes to which the boundary will be applied.
+
+        lon_range (:class:'tuple'):
+            The two-tuple containting the start and end of the desired range of
+            longitudes. The first entry must be smaller than the second entry.
+            Both entries must be between [-180 , 180].
+
+        lat_range (:class:'tuple'):
+            The two-tuple containting the start and end of the desired range of
+            longitudes. The first entry must be smaller than the second entry.
+            Both entries must be between (-90 , 90).
+
+        res (:class:'int'):
+            The size of the incrementation for vertices in degrees. Default is
+            a vertex every one degree of longitude.
+
+    """
+
+    # Set extent of map
+    ax.set_extent([lon_range[0], lon_range[1], lat_range[0], lat_range[1]],
+                  ccrs.PlateCarree())
+    # Make a boundary path in PlateCarree projection begining in the bottom
+    # left and continuing anitclockwise creating a point every `res` degree
+    vertices = [(lon, lat_range[0]) for lon in range(lon_range[0], lon_range[1] + 1, res)] + \
+               [(lon, lat_range[1]) for lon in range(lon_range[1], lon_range[0] - 1, -res)]
+    boundary = mpath.Path(vertices)
+    ax.set_boundary(boundary, transform=ccrs.PlateCarree())
+
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into
+# xarrays and disable time decoding due to missing necessary metadata
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+# Extract a slice of the data
+ds = ds.isel(time=0).drop("time")
```

Comment:
There is a future deprecation warning for using `drop` function for dropping labels and it suggests to use `drop_sel` instead if possible. Please try to see the deprecation warning on your own and it would be good to avoid such a deprecation before merging the code.

Reply:
`drop_vars` is a function recommended on the xarray page for `drop`. I have used that instead and it works.